molecule/common/prepare.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: "Add config.json for collectd test container"
+  hosts: collectd-test
+  tasks:
+    - name: "Make sure that /var/lib/kolla/config_files exists"
+      file:
+        path: "/var/lib/kolla/config_files"
+        state: directory
+
+    - name: "Add config.json"
+      copy:
+        src: "{{ playbook_dir }}/../../tests/kolla_config_files/config.json"
+        dest: /var/lib/kolla/config_files/config.json
```

Comment:
I assume this is the #208 violation? Any reason you don't just add the required perms?

Reply:
This is possibly going away. I'm considering whether it's a good idea to run the deploy_steps_tasks and create the configs on the host, then mount them into the container. However, there are some side effects to running the tests, namely that there's some additional setup tasks taking place in [deploy_steps_tasks.yml ](https://github.com/infrawatch/tripleo-collectd-ansible-role/blob/molecule_tests/tasks/deploy_steps_tasks.yml)(which is called from tripleo_collectd/tasks/main.yml, using the deploy_stage var), and this is what is run by molecule.

Reply:
I am planning on separating the "configure_collectd" task out of deploy_steps_tasks.yml, so that it can be moved to a different stage later, if required (it'll also make it easier to test this action on its own instead of within a deploy step with other actions run as part of the same step.